### PR TITLE
new package: libmuparser

### PR DESCRIPTION
**Pull request by pvonmoradi**
Adds [muparser](https://beltoforion.de/en/muparser/) library for parsing math expressions.

### CODE REVIEW

File: packages/libmuparser/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE="https://beltoforion.de/en/muparser"
+TERMUX_PKG_DESCRIPTION="An extensible high performance math expression parser \
+library written in C++"
```
Comment (Grimler91) [inline packages/libmuparser/build.sh:3]: Why line break? Will break something or at least look weird on device

File: packages/libmuparser/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```
Comment (Grimler91) [inline packages/libmuparser/build.sh:6]: This line shouldn't be needed, LICENSE is one of the filenames [termux_step_install_license looks for](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L35-L38)

File: packages/libmuparser/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="2.3.3-1"
+TERMUX_PKG_SRCURL="https://github.com/beltoforion/muparser/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=91d26d8274ae9cd9c776ee58250aeddc6b574f369eafd03b25045b858a2b8177
+TERMUX_PKG_BUILD_DEPENDS="cmake"
```
Comment (Grimler91) [inline packages/libmuparser/build.sh:11]: This line is not needed, cmake is installed by build system automatically. `TERMUX_PKG_BUILD_DEPENDS` means that it downloads cmake that has been cross-compiled for termux, and that cmake cannot be used in any case

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Comment by truboxl** [comment]
Please add `TERMUX_PKG_DEPENDS="libc++"`

**Pull request merged**